iop/fs/Makefile

```diff
@@ -10,8 +10,8 @@ SUBDIRS = \
 	adddrv \
 	addrom2 \
 	bdm \
-	bdmfs_fatfs \
```

Comment:
Shouldn't we just have here the `bdm` and then inside the `bdm` folder to compile each of the subfolders?

Reply:
Hey there i was thinking that the irx compiled would be the same as bdmfs_fatfs, i plan to find a fix to this one or rename back to `bdmfs_fatfs`

Reply:
@fjtrujy fixed: https://github.com/ps2dev/ps2sdk/pull/515/commits/a59626e385c28f597dcced2bbb0c3938ee0aa6d2

Reply:
Maybe I didn't explain properly before, I wanted to say that I would expect to have in this file just "bdm" right now we still have `bdm/bdm_fatfs` and `bdm/bdm_vfat`.
Those "subfolders" compilation should be done in the `bdm/Makefile` file...

Reply:
> Maybe I didn't explain properly before, I wanted to say that I would expect to have in this file just "bdm" right now we still have `bdm/bdm_fatfs` and `bdm/bdm_vfat`. Those "subfolders" compilation should be done in the `bdm/Makefile` file...

I think it shoudln't affect the subfolders compilation, but i'm thinking that you want to remove these `bdm/bdm_fatfs` and `bdm/bdm_vfat` make comands and add only bdm. Sounds fine to me.